### PR TITLE
Fix 10642: Mark delete entities and tags toggle

### PR DESCRIPTION
**Pull request by OnkarVO7**
### Describe your changes :

Fix https://github.com/open-metadata/OpenMetadata/issues/10642
Fix https://github.com/open-metadata/OpenMetadata/issues/10714

#
### Type of change :

- [x] New feature

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@open-metadata/ingestion

### CODE REVIEW

File: ingestion/setup.py [hunk 1]
```diff
@@ -179,7 +179,7 @@ def get_long_description():
     "kinesis": {VERSIONS["boto3"]},
     "ldap-users": {"ldap3==2.9.1"},
     "looker": {"looker-sdk>=22.20.0"},
-    "mlflow": {"mlflow-skinny~=1.30"},
+    "mlflow": {"mlflow-skinny~=1.30", "alembic==1.10.2"},
```
Comment (OnkarVO7) [inline ingestion/setup.py:182]: `alembic` was giving not found error for mlflow. Added it here in case it was missed

**Review by NiharDoshi99 — DISMISSED** [review]

File: ingestion/setup.py (hunk 1 above)
Comment (pmbrull) [reply ingestion/setup.py:182]: can we keep this a bit laxer? `~=`

Comment (OnkarVO7) [reply ingestion/setup.py:182]: Sure I'll do that

File: openmetadata-spec/src/main/resources/json/schema/metadataIngestion/dashboardServiceMetadataPipeline.json [hunk 2]
```diff
@@ -35,6 +35,16 @@
...
       "type": "boolean",
       "default": "false"
+    },
+    "markDeletedDashboards": {
+      "description": "Optional configuration to soft delete dashboards in OpenMetadata if the source dashboards are deleted. Also, if the dashboard is deleted, all the associated entities like lineage, etc., with that dashboard will be deleted",
+      "type": "boolean",
+      "default": true
+    },
+    "includeTags": {
+      "description": "Optional configuration to turn off fetching metadata for tags.",
```
Comment (pmbrull) [inline openmetadata-spec/src/main/resources/json/schema/metadataIngestion/dashboardServiceMetadataPipeline.json:45]: maybe something like "to toggle tags ingestion"

**Review by chirag-madlani — DISMISSED** [review]

**Review by NiharDoshi99 — APPROVED** [review]

**Review by NiharDoshi99 — DISMISSED** [review]

**Review by ayush-shah — APPROVED** [review]
Approved from the github workflow and cypress changes related to taxonomy project id

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**